src/middlewared/middlewared/plugins/rsync.py

```diff
@@ -289,15 +292,31 @@ class Config:
         Int('maxconn'),
         Str('user'),
         Str('group'),
-        Str('hostsallow'),
-        Str('hostsdeny'),
+        List('hostsallow', items=[Str('hostsallow')]),
+        List('hostsdeny', items=[Str('hostdeny')]),
         Str('auxiliary'),
         register=True,
     ))
     async def do_create(self, data):
+        if re.search(r'[/\]]', data.get('name')):
```

Comment:
If you implement this as `Str('name', validators=[Match(r'[^/\]]')])`, this particular validation error won't stop other fields from validating. We try to avoid raising `ValidationError` directly in the method for that reason.